xgcm/grid.py

```diff
@@ -1225,9 +1225,9 @@ def __init__(
 
     def _as_axis_kwarg_mapping(
         self,
-        kwargs: Union[str, float, int, Dict[str, Union[str, float, int]]],
-        axes: Optional[list] = None,
-    ) -> Dict[str, Union[str, float, int]]:
+        kwargs: Union[Any, Dict[str, Any]],
+        axes: List[str] = None,
```

Comment:
I guess this could be a tuple too? I was never clear on that. But it might be good if we try to enforce one one type here?

Reply:
What could be a tuple? The list? In which case we should put Iterable there instead.